app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
```

Comment:
I think we've got a couple of these twice. It doesn't cause any problems, but we should probably get rid of them at some point